ply-core/src/Ply.hs

```diff
@@ -19,11 +19,19 @@ module Ply (
 
 import Data.Coerce (coerce)
 
+#if PLUTUS_NEW
 import PlutusLedgerApi.V1.Scripts (
   MintingPolicy (MintingPolicy),
   Script (Script),
   Validator (Validator),
  )
+#else
+import Plutus.V1.Ledger.Scripts (
+  MintingPolicy (MintingPolicy),
+  Script (Script),
+  Validator (Validator),
+ )
+#endif
```

Comment:
A little thing we could do is have a local module called `LedgerExports` where all the necessary ledger stuff are exported and CPP is used. Mostly, it should just export the `PlutusLedgerApi.V1` and a few stuff from `.Scripts` - nothing else will be necessary I presume.

All other modules can use this local module without touching hideous CPP!

Reply:
I've add a module for this, let me know if there's anything else that can be improved here.

Would it be worth having modules for different ledger API versions?

Reply:
Yes, in the future - certainly. It's fine for now though. I'll add that during #10